draft-ietf-quic-transport.md

```diff
@@ -364,6 +364,8 @@ keys are established.
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                       Packet Number (32)                      |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|     [Payload Length (16)]     |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                          Payload (*)                        ...
```

Comment:
Totally editorial:  You could put Payload in the second half of the previous line to avoid having a jagged illustration.  Variable-length fields like Payload don't have to have any particular width.

Reply:
Thank you very much for the review. I've fixed this in a52715a. I will address other issues later (hopefully tomorrow).